templates/samples/statusBar/index.ts

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
we've to avoid upppercase in folder names

samples/statusBar --> samples/status-bar